projects/efg-training/resources/test/app-subnets.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_subnet" "efg_app_1" {
+    vpc_id            = "${aws_vpc.vpc.id}"
```

Comment:
Since we have multiple VPCs and are not creating a new one here, will this return the default VPC?